src/index.d.ts

```diff
@@ -1,6 +1,6 @@
 declare module "local-devices" {
 
-    function findLocalDevices(address?: any): Promise<findLocalDevices.IDevice[]>;
+    function findLocalDevices(address?: any, skipNameResolution?: boolean, arpPath?: string): Promise<findLocalDevices.IDevice[]>;
```

Comment:
```suggestion
    function findLocalDevices(address?: any, { skipNameResolution?: boolean, arpPath?: string }): Promise<findLocalDevices.IDevice[]>;
```

I vote for adding the additional parameters in an `options` object. This allows us to add more flags, options, etc. in the future without breaking the API or extending the list of parameters much longer.

@DylanPiercey This will be breaking, _but_ will allow us to be more flexible in the future!

### Example

```js
const foo = findLocalDevices('192.168.0.10', {skipNameResolution: false, arpPath: 'arp'})
```

Two more things

- take care in the test that this is tested
- update the README accordingly.

Thanks.

PS: please change the type of `address` to `address?: string | null`.

Reply:
Yeah I agree.

Reply:
Definitely agree on this - it's a breaking change, but it makes a lot of sense once you go beyond a couple of arguments.

I've implemented the proposed change, and actually gone a bit further - I've moved the `address` argument into the object and renamed it to `ipAddressOrRange`. This feels like the right move to me - it's weird to have an optional (nullable?) positional argument followed by an object.

How do these updates look to you? 😊

Reply:
How about just `address` for the name? `ipAddressOrRange` seems a bit verbose.

Reply:
I'm cool with that. We can use the description to make it clear that it can be a single address or a range.

Reply:
@timrogers if you want to go ahead with that then that'd be great. I think otherwise I'm happy with the changes!